src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerTypeSymbol.cs

```diff
@@ -12,19 +12,21 @@
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     // PROTOTYPE(func-ptr): Support generic substitution and retargeting
-    internal sealed class FunctionPointerTypeSymbol : TypeSymbol
+    // PROTOTYPE(func-ptr): Match msvc's emitting of custom modifiers for calling convention
+    internal sealed partial class FunctionPointerTypeSymbol : TypeSymbol
     {
         public static FunctionPointerTypeSymbol CreateFunctionPointerTypeSymbolFromSource(FunctionPointerTypeSyntax syntax, Binder typeBinder, DiagnosticBag diagnostics, ConsList<TypeSymbol> basesBeingResolved, bool suppressUseSiteDiagnostics)
```

Comment:
>FunctionPointerTypeSymbolFromSource [](start = 54, length = 35)

Consider shortening name to `Create` or `CreateFromSource`. Similar comment for `Create...FromMetadata` below.